### DISCUSSION TITLE
Is there a way to reflect hmr in nextjs while configuring Turborepo with panda.css?

### DISCUSSION THREAD

**Discussion by skulter** [chakra-ui/panda#2791] [Q&A]
I currently have a monorepo configured with TurboRepo, my project exists in apps and I have a ui folder configured in packages.
I have the dependency "@pandacss/dev" installed in the global package.json.

If this is the case, the project consisting of react + vite located in apps will trigger HMR when modifying the configuration recipe of panda.css, so you can check it in real time whenever you modify the recipe.
The project consisting of nextjs cannot be checked in real time.

To solve this, execute "prepare": "panda codegen -watch" to the ui folder, but when the recipe of the ui folder is modified, NEXTJS generates the following error and does not recover anymore. Is there any workaround?
-----
Module not found: Can't resolve '../helpers.mjs'
> 1 | import { memo, splitProps } from '../helpers.mjs';
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  2 | import { createRecipe, mergeRecipes } from './create-recipe.mjs';
  3 |
  4 | const selectTriggerFn = /* @__PURE__ */ createRecipe('select-trigger', {}, [])
----
my project is assigned a recipe from package/ui as a preset.

**Comment by segunadebayo** [comment]
Providing a reproduction is the best way to get help. Could you send over a repo/codesandbox?
